Source/Core/DolphinQt/NetPlay/PadMappingDialog.cpp

```diff
@@ -3,15 +3,17 @@
 // Refer to the license.txt file included.
 
 #include "DolphinQt/NetPlay/PadMappingDialog.h"
-#include "DolphinQt/Settings.h"
-
-#include "Core/NetPlayClient.h"
 
 #include <QComboBox>
 #include <QDialogButtonBox>
 #include <QGridLayout>
 #include <QLabel>
 
+#include "Core/NetPlayClient.h"
+#include "Core/NetPlayServer.h"
+
+#include "DolphinQt/Settings.h"
```

Comment:
Any reason you moved this include down here?

Reply:
Yes. It doesn't follow our coding style. Only the header for the source file should be at the top, then it's supposed to follow:

```
<standard includes>

<library includes>

<dolphin includes>
```

Reply:
i.e. see how MD5Dialog is laid out (that same include is in the correct place).

Reply:
Ah, I missed the fact that the one above it previous was the header include.